src/ImageReader.cpp

```diff
@@ -100,17 +97,19 @@ std::shared_ptr<Frame> ImageReader::GetFrame(int64_t requested_frame)
 {
 	// Check for open reader (or throw exception)
 	if (!is_open)
-		throw ReaderClosed("The FFmpegReader is closed.  Call Open() before calling this method.", path);
+		throw ReaderClosed(
+            "The FFmpegReader is closed. "
+            "Call Open() before calling this method.", path);
```

Comment:
Ahhh, crap. Not only did I screw up the indentation here, but I failed to notice it's calling itself the _wrong_ class.